### ISSUE TITLE
Backend: QR Code generation

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#47]
## User story
1. As a HR employee
2. I want / need the backend to create a QR code 
3. So that the QR code can be send out to user 

## Acceptance criteria
* Transform URL to QR code

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Jean28518** [comment]
Would be great if you can finish this until friday evening

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272

**Comment by TamaraHoock** [comment]
Figma
    
    
    
    
    
    
    
    
  /*<![CDATA[*/
  :root {
    color-scheme: light dark;
    supported-color-schemes: light dark;
  }
  /*]]>*/
  
    
  /*<![CDATA[*/
  @font-face {
    font-family: 'Whyte';
    font-style: normal;
    font-weight: normal;
    src: /*url(.eot?#iefix) format('embedded-opentype'),
    url(.woff2) format('woff2'),
    url(.ttf) format('truetype'),*/
    url(https://static.figma.com/uploads/9254519927cf4912e61627442d7ab7a5a9226581.woff) format('woff');
    mso-font-alt: 'Arial';
  }

  h1{font-size: 26px; line-height:32px; font-family:'Whyte', Helvetica, Arial, sans-serif; color:#000000; text-align: center; font-weight: normal; margin: 0;}
  h2{font-size: 20px; line-height:28px; font-family:'Whyte', Helvetica, Arial, sans-serif; color:#000000; text-align: center; font-weight: normal;margin: 0;}
  h3{font-weight:bold; font-size: 20px; line-height: 28px; font-family:'Whyte', Helvetica, Arial, sans-serif; color:#000000; text-align: left;margin: 0;}
  h4{font-size:18px; line-height: 24px; font-family:'Whyte', Helvetica, Arial, sans-serif; color:#333333; font-weight: normal;margin: 0;}
  h5{font-size: 36px; font-weight:bold; line-height: 42px; font-family:'Whyte', Helvetica, Arial, sans-serif; color:#000000; margin: 0;}
  h6{font-weight:bold; font-size: 32px; line-height: 38px; font-family:'Whyte', Helvetica, Arial, sans-serif; color:#000000; letter-spacing:-1px; margin: 0;}
  sup{font-size: 12px; color: #9747FF; font-weight: bold; font-family:'Whyte', Helvetica, Arial, sans-serif; vertical-align: top; line-height: 0; mso-font-size: 12px !important; mso-text-raise:40%;}

  a{outline:none;color:#000; text-decoration:none;}
  a:hover{text-decoration:none !important;}
  .h-u a{text-decoration:underline;}
  .h-u a:hover, .active-t a:hover{text-decoration:none !important;}
  a[x-apple-data-detectors]{color:inherit !important; text-decoration:none !important;}
  .active-i a:hover,
  .active-t:hover{opacity:0.8;}
  .active-i a,
  .active-t{transition:all 0.3s ease;}
  .link:hover { text-decoration:none !important; }
  .cta-border:hover { border-bottom: 1px solid transparent !important; }
  }
  .cta { transition: 0.3s !important; }
  a img{border:none; outline: none; text-decoration: none;}
  b, strong{font-weight:700;}
  p{margin:0;}
  th{padding:0;}
  table, td{mso-table-lspace:0; mso-table-rspace:0; border-spacing: 0; border: 0;}
  .ns span,
  .ns a{color:inherit !important; text-decoration:none !important; border:none !important;}

  @media only screen and (max-width:375px) and (min-width:320px) {
    .gmail-fix{min-width:320px !important;}
  }
  @media only screen and (max-width:414px) and (min-width:413px) {
    .gmail-fix{min-width:413px !important;}
  }

  @media screen and (max-width: 700px) {
    .w-100p{width:100%!important;}
    .wi-100p img{width:100%!important;height:auto!important;}
    .wi-33p img{width:33%!important;height:auto!important;}
    .tflex{display:block!important;width:100%!important;}
    .thold{display:table!important;width:100%!important;}
    .trow{display:table-row!important;width:100%!important;}
    .tfoot{display:table-footer-group!important;width:100%!important;}

    .fs-18{font-size:18px!important;}
    .fs-22{font-size:22px!important;}
    .fs-32{font-size:32px!important;}
    .lh-38{line-height: 38px!important;}
    .lh-26{line-height: 26px!important;}

    .pt-0{padding-top:0!important;}
    .pt-24{padding-top:24px!important;}
    .pt-40{padding-top:40px!important;}
    .pt-60{padding-top:60px!important;}
    .pt-70{padding-top:70px!important;}

    .pb-0{padding-bottom:0!important;}
    .pb-24{padding-bottom:24px!important;}
    .pb-40{padding-bottom:40px!important;}
    .pb-50{padding-bottom:50px!important;}
    .pb-60{padding-bottom:60px!important;}
    .pb-70{padding-bottom:70px!important;}

    .pl-0{padding-left:0px!important;}
    .pl-20{padding-left:20px!important;}
    .pl-35{padding-left:35px!important;}

    .pr-0{padding-right:0px!important;}
    .pr-20{padding-right:20px!important;}
    .pr-35{padding-right:35px!important;}

    .plr-15{padding-left:15px!important;padding-right:15px!important;}
    .plr-25{padding-left:25px!important;padding-right:25px!important;}

    .ptb-24{padding-top:24px!important;padding-bottom:24px!important;}
    .ptb-45{padding-top:45px!important;padding-bottom:45px!important;}
    .ptb-60{padding-top:60px!important;padding-bottom:60px!important;}
  }
  /*]]>*/
  
    
  /*<![CDATA[*/
  @media (prefers-color-scheme: dark) {

    /* Shows Dark Mode-Only Content, Like Images */
    .dark-img {
      display: block !important;
      width: auto !important;
      overflow: visible !important;
      float: none !important;
      max-height: inherit !important;
      max-width: inherit !important;
      line-height: auto !important;
      margin-top: 0px !important;
      visibility: inherit !important;
    }

    /* Hides Light Mode-Only Content, Like Images */
    .light-img {
      display: none;
      display: none !important;
    }

    /* Custom Dark Mode Background Color */
    .darkmode {
      background-color: #000000 !important;
    }

    /* Custom Dark Mode Font Colors */
    h1, h2, h3, h4, h5, h6, p, ul, li {
      color: #fdfdfd !important;
    }
    span {
      color: #000000 !important;
    }
    a {
      color: #fdfdfd !important;
    }


    /* Custom Dark Mode Text Link Color */
    .link { color: #ffffff !important; }
    .footer a { color: #fdfdfd !important; }
    .cta-border { color: #fdfdfd !important; border: 3px solid #fdfdfd !important; }
    .cta-border span { color: #7EABCE !important; }
    .btn-black { background-color: #fdfdfd !important; color: #000000 !important; border: 3px solid #fdfdfd !important; border-radius: 5px !important; mso-color: #000000 !important; mso-border: 3px solid #fdfdfd !important; mso-border-radius: 5px !important;}
    .btn-white { background-color: #fdfdfd !important; color: #000000 !important; border: 3px solid #fdfdfd !important; border-radius: 5px !important; mso-color: #000000!important; mso-border: 3px solid #fdfdfd !important; mso-border-radius: 5px !important;}
    .btn-white2 { background-color: #000000 !important; color: #ffffff !important; border: 3px solid #fdfdfd !important; border-radius: 5px !important; mso-color: #ffffff !important; mso-border: 3px solid #fdfdfd !important; mso-border-radius: 5px !important; }
  }

  /* Copy dark mode styles for android support */
  /* Shows Dark Mode-Only Content, Like Images */
  [data-ogsc] .dark-img {
    display: block !important;
    width: auto !important;
    overflow: visible !important;
    float: none !important;
    max-height: inherit !important;
    max-width: inherit !important;
    line-height: auto !important;
    margin-top: 0px !important;
    visibility: inherit !important;
  }

  /* Hides Light Mode-Only Content, Like Images */
  [data-ogsc] .light-img {
    display: none;
    display: none !important;
  }

  /* Custom Dark Mode Background Color */
  [data-ogsc] .darkmode {
    background-color: #000000 !important;
  }

  [data-ogsc] .darkmode2, [data-ogsc] .callout {
    background-color: #1c1c1c !important;
  }

  /* Custom Dark Mode Font Colors */
  [data-ogsc] h1, [data-ogsc] h2, [data-ogsc] h3, [data-ogsc] h4, [data-ogsc] h5, [data-ogsc] h6, [data-ogsc] p, [data-ogsc] li, [data-ogsc] ul {
    color: #fdfdfd !important;
  }
  [data-ogsc] span {
    color: #000000 !important;
  }
  [data-ogsc] a {
    color: #000000 !important;
  }

  /* Custom Dark Mode Text Link Color */
  [data-ogsc] .link { color: #ffffff !important; }
  [data-ogsc] .footer a { color: #fdfdfd !important; }
  [data-ogsc] .cta-border { color: #7EABCE !important; border-bottom: 3px solid #7EABCE !important; }
  [data-ogsc] .cta-border span { color: #7EABCE !important; }
  [data-ogsc] .btn-black { background-color: #fdfdfd !important; color: #000000 !important; border: 3px solid #fdfdfd !important; border-radius: 5px !important; mso-background-color-alt: #fdfdfd !important; mso-color-alt: #000000 !important; mso-border-alt: 3px solid #fdfdfd !important; mso-border-radius-alt: 5px !important;}
  [data-ogsc] .btn-white { background-color: #fdfdfd !important; color: #000000 !important; border: 3px solid #fdfdfd !important; border-radius: 5px !important; mso-background-color-alt: #fdfdfd !important; mso-color-alt: #000000!important; mso-border-alt: 3px solid #fdfdfd !important; mso-border-radius-alt: 5px !important;}
  [data-ogsc] .btn-white2 { background-color: #000000 !important; color: #ffffff !important; border: 3px solid #fdfdfd !important; border-radius: 5px !important; mso-color-alt: #ffffff !important; mso-border-alt: 3px solid #fdfdfd !important; mso-border-radius-alt: 5px !important; mso-background-color-alt: #000000 !important;}
  /*]]>*/
  
  
  
  
  
  
  
  
  
  
  
  
  
  
    
       Tamara H has invited you to join the Team "DIdentity"  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ 
    
      
        
        
          
            
              
                
                  
                     
                        
                        
                          
                        
                         

                  
                    
                      
                        

                          
                             Tamara H has invited you to join the Team "DIdentity"
                            
                          


                          
                            
                              
                                
                                  
                                     Accept invite
                                  
                                
                               
                          
                          
                       
                  
                          
                          
                            
                              
                                
                                  
                                    Figma is a design platform for teams who build products together. Born on the Web, Figma helps the entire product team create, test, and ship better designs, faster.
                                  
                                
                               
                          
                          
                            
                              
                                
                                  
                                     
                                    
                                     
                                    
                                     
                                    
                                     
                                    
                                     
                                    
                                     
                                  
                                
                               
                          
                          
                            760 Market Street, Floor 10 San Francisco, CA, 94102
